xdsl/dialects/riscv.py

```diff
@@ -447,7 +446,7 @@ def print_op_type(self, printer: Printer) -> None:
 
 
 AssemblyInstructionArg: TypeAlias = (
-    IntegerAttr | LabelAttr | SSAValue | IntRegisterType | str
+    IntegerAttr | LabelAttr | SSAValue | RegisterType | str
```

Comment:
Right, is this only used in this file? (Just want to check that this isn't used in an `operand_def` or something)